### ISSUE TITLE
Temadager

### ISSUE THREAD

**Issue by henrist** [blindernuka/billett#79]
Liste over datoer og tilhørende tema/tittel. Bør kunne legge til ekstra tekst som vises på arrangementsiden.

- [ ] Legge til
- [ ] Redigere
- [ ] Slette
- [ ] Vise på arrangementliste (gjester)
- [ ] Vise på admin side (arrangementliste)

**Comment by henrist** [comment]
Foreslår at dette bare legges inn som data i en egen kolonne i arrangementgruppen. Kan lagres som json av praktiske årsaker.

Forslag:
```json
[
  {"day": "xxxx-xx-xx", "title": "xxx", "description": "description with markdown"},
  {"day": "xxxx-xx-xx", "title": "xxx", "description": "description with markdown"}
]
```

**Comment by vegarda** [comment]
@henrist Lyst å legge til temadager?

**Comment by henrist** [comment]
Ja, men vet ikke om jeg får tid nå med det første.. Det er vel bare hardkodet per i dag, right?

**Comment by vegarda** [comment]
@henrist Ok. Riktig, jeg la inn temadager for miniuka 2016

**Comment by henrist** [comment]
Relevante commits:
https://github.com/blindernuka/billett-frontend/commit/c8840462d06cefa3892470d0445f26c60489e79d
https://github.com/blindernuka/billett/commit/54eba5633a99dd127d224781f370ba800db1f93e
https://github.com/blindernuka/billett/commit/0c87e8936d67acf4971637b180b51f08b4709bd7
